crates/hyperswitch_connectors/src/connectors/santander.rs

```diff
@@ -279,26 +300,49 @@ impl ConnectorIntegration<Authorize, PaymentsAuthorizeData, PaymentsResponseData
         req: &PaymentsAuthorizeRouterData,
         connectors: &Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
-        Ok(format!(
-            "{}cob/{}",
-            self.base_url(connectors),
-            req.payment_id
-        ))
+        let santander_mca_metadata =
+            santander::SantanderMetadataObject::try_from(&req.connector_meta_data)?;
+        match req.request.payment_method_type {
+            Some(enums::PaymentMethodType::Pix) => Ok(format!(
+                "{}cob/{}",
```

Comment:
can you please verify if the url correct or should it be like the following instead?
```suggestion
                "{}/cob/{}",
```

Reply:
the first `/` is not needed because the base url itself has a `/` at the last